app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/snippet/CreateSnippetConverter.kt

```diff
@@ -0,0 +1,20 @@
+package com.bottlerocketstudios.brarchitecture.ui.snippet
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import com.bottlerocketstudios.compose.snippets.CreateSnippetScreenState
+
+@Composable
+fun CreateSnippetFragmentViewModel.toState() = CreateSnippetScreenState(
+    title = title.collectAsState(),
+    filename = filename.collectAsState(),
+    contents = contents.collectAsState(),
+    private = private.collectAsState(),
```

Comment:
@VanyaHuamanBottleRocket Should we refactor this to `isPrivate`?

cc: @BottleRocket-Colin

Reply:
Renaming